packages/node_modules/@ciscospark/react-redux-spark/src/spark.js

```diff
@@ -71,8 +71,13 @@ function defaultConfig(options = {}) {
     },
     credentials: {
       client_id: process.env.CISCOSPARK_CLIENT_ID,
+      federation: true,
       scope: 'spark:all spark:kms'
     },
+
+    // Federation EU cluster
+    orgId: process.env.EU_PRIMARY_ORG_ID,
```

Comment:
I'm a bit confused by this env name. Are federation org ids all going to be "EU"?